plugin/pkg/scheduler/algorithm/priorities/taint_toleration.go

```diff
@@ -52,8 +52,8 @@ func getAllTolerationPreferNoSchedule(tolerations []v1.Toleration) (tolerationLi
 	return
 }
 
-func getTolerationListFromPod(pod *v1.Pod) ([]v1.Toleration, error) {
```

Comment:
@aveshagarwal @bsalamat, I have updated the code as per your suggestions. I also modified some parts of code as per go standards. Please let me know if any other changes are needed.